packages/forma-36-react-components/src/components/Textarea/README.mdx

```diff
@@ -4,17 +4,56 @@ type: 'component'
 status: 'stable'
 slug: /components/textarea/
 github: 'https://github.com/contentful/forma-36/tree/master/packages/forma-36-react-components/src/components/Textarea'
-storybook: 'https://f36-storybook.contentful.com/?path=/story/components-textarea--default'
+storybook: 'https://f36-storybook.contentful.com/?path=/story/form-elements-textarea--basic'
 ---
 
-Detailed documentation will be provided soon. For now, please have a look into the storybook for implementation details.
+`Textarea` is a form element that allows the user to enter a sizeable amount of multi-line plain text.
+
+## Table of contents
+
+- [How to use Textarea](#how-to-use-Textarea)
+- [Code examples](#code-examples)
+- [Accessibility](#accessibility)
+
+## How to use Textarea
+
+Textarea is a form element, it should be used within the context of `<Form>`
+It can be configured in a number of different ways; use this guide to determine the best configuration based on the context.
+
+## Code examples
+
+```tsx
+// import { Textarea } from '@contentful/forma-36-react-components';
 
-```jsx
 <Textarea
-  name="someInput"
-  id="someInput"
+  name="textarea"
+  id="textarea"
```

Comment:
Maybe a more semantic value would make this even more helpful, like "description" or "story" or something else that hints at a long text?